serving/ingress/pkg/controller/ingress/resources/route.go

```diff
@@ -91,12 +91,9 @@ func makeRoute(ci *networkingv1alpha1.Ingress, host string, rule networkingv1alp
 		}
 	}
 
-	labels := make(map[string]string)
-	labels[networking.IngressLabelKey] = ci.GetName()
-
-	ingressLabels := ci.GetLabels()
-	labels[serving.RouteLabelKey] = ingressLabels[serving.RouteLabelKey]
```

Comment:
Just a note about this deletion, Ingress has the label https://github.com/knative/serving/blob/b1a021dfcf522d5c7e2d7513501696a07e7a25d0/pkg/reconciler/route/resources/ingress.go#L72-L75
so we don't need to copy it one by one.